test/xbgpu/test_engine.py

```diff
@@ -271,12 +271,12 @@ def valid_end_to_end_combination(combo: dict) -> bool:
 def verify_corrprod_sensors(
     xpipelines: list[XPipeline],
     corrprod_results: list[np.ndarray],
-    missing_antenna: int | None,
-    incomplete_accumulation_counters: list[int],
     prom_diff: PromDiff,
+    actual_sensor_updates: dict[str, list[tuple[Any, aiokatcp.Sensor.Status]]],
+    incomplete_accumulation_counters: list[int],
     n_channels_per_substream: int,
     n_baselines: int,
-    actual_sensor_updates: dict[str, list[tuple[Any, aiokatcp.Sensor.Status]]],
+    missing_antenna: int | None,
```

Comment:
Yeah apologies for the seemingly unnecessary reshuffle of the arglist. Something just wasn't sitting right about the order.